README.md

```diff
@@ -1 +1,155 @@
-# desi-dhaba-js-capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
```

Comment:
- [ ] Wow, your project is really impressive, especially since you did it all by yourself! Awesome work! Just a small note: it seems the video link is missing in your PR description. Would you mind adding it? Thank you!

   ![image](https://github.com/Arslan49sp/desi-dhaba-js-capstone/assets/107952724/0f09446b-7891-4d88-9dd4-da5b1fa4bdd8)